python/tvm/relay/expr.py

```diff
@@ -98,39 +125,87 @@ def __init__(self,
 
 @register_relay_node
 class Call(Expr):
-    """A function call in Relay, see tvm/relay/expr.h for more details."""
+    """Function call node in Relay.
+
+    Call node corresponds the operator application node
```

Comment:
"The call node corresponds to the operator application node in computational graph terminology."